### PR TITLE
[PC-1178] Portenta H7 Family Tech Specs Update

### PR DESCRIPTION
**Pull request by canchebagur**
## What This PR Changes
- This PR changes the tech specs tables of the Portenta H7 family boards, standardizing them with the Portenta C33 board. 

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by jcarolinares — CHANGES_REQUESTED** [review]
Please @GDJose help me to confirm some of this data with the sources. We want to be sure between all of us that we are not making mistakes.

Review only done at this moment for Portenta H7

File: content/hardware/04.pro/boards/portenta-h7/tech-specs.yml [hunk 1]
```diff
@@ -1,27 +1,59 @@
...
+  SPI pins: 4 
+  I2C pins: 6
+  UART pins: 16
+  CAN pins: 4
+  I2S pins: 4
+  USB: 6
+  JTAG/SWD: 4
+MKR-styled pins:
+  Digital I/O pins: 22
+  Analog pins: 8
```
[the remark this answers is not in the document]
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml:19]: Pins with analog functionality are 8, AREF is an analog pin. Analog **input** pins are 7.

File: content/hardware/04.pro/boards/portenta-h7/tech-specs.yml [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
+  Channels: 1
+  Resolution: 12 bits 
+Communication:
+  I2C: 3
+  UART: 4
+  CAN: 2
+  I2S: 1
+  SPI: 2
+  PWM: 10
+  USB: USB 1.0/Full-Speed (x1) and USB 2.0/Hi-Speed (x1)
```
[the remark this answers is not in the document]
Comment (canchebagur) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml:45]: Double-checking the Altium files I found that only one SPI port is accessible to the user via the HD connectors and MKR-styled pins. Regarding the USB, the microcontroller datasheet states that one of the USB ports has double functionality, FS and HS, but in the schematics, it is shown explicitly as HS. What should we do in that case @jcarolinares? I initially put just the two ports that are shown in the schematic, FS and HS.

[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml]: Let's keep it this way as we already talked during our 1x1

File: content/hardware/04.pro/boards/portenta-h7/tech-specs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml:19]: As we already talk during oour one to one we will say Analog input pins and put 7

**Comment by canchebagur** [comment]
@jcarolinares all of your comments and observations were addressed in the last commit, it should be ready now. Let me know your thoughts about it.

File: content/hardware/04.pro/boards/portenta-h7/tech-specs.yml [hunk 3]
```diff
@@ -1,27 +1,59 @@
...
+Wireless connectivity:
+  Chipset: Murata® LBEE5KL1DX
+  Wi-Fi®: 2.4 GHz 802.11 b/g/n
+  Bluetooth®: Bluetooth® 4.1
+Ethernet connectivity:
+  Chipset: LAN8742AI
+  Interface and rates: RMII 10/100 Mbps
+Secure element: NXP® SE050C2 and Microchip® ATECC608
+Analog-to-digital converter:
+  Channels: 8
```
[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml:33]: @canchebagur please re-check this data, If I'm not wrong this should be 7 ???

[the remark this answers is not in the document]
Comment (jcarolinares) [reply content/hardware/04.pro/boards/portenta-h7/tech-specs.yml:33]: Perfect

Solved.

**Comment by jcarolinares** [comment]
thank you @TaddyHC for the nice trigger trick

**Comment by jcarolinares** [comment]
Additional feedback from @martab1994 

Changes done and double-checked with Gatbsy Preview

Merging

**Pull request merged**